gulpfile.js

```diff
@@ -233,20 +30,7 @@ gulp.task('default', function (done) {
   done();
```

Comment:
@theresaanna 
Let's remove the `[ flags, * ]` from this section in this PR since they're commented it out for now. I think more PRs can come after this one to add specific functionality like `scripts`, `production`, `no-test`, etc.

```
   gutil.log('$', gutil.colors.magenta('gulp [ production, no-test ] build'));
   gutil.log('Build the asset-pipeline with optional production and no-test flags');
   gutil.log('$', gutil.colors.magenta('gulp clean-all'));
   gutil.log('Removes files and directories generated by gulp');
   gutil.log('$', gutil.colors.magenta('gulp [ production ] website'));
   gutil.log('Runs the gulp watch and hugo serve');
   gutil.log('$', gutil.colors.magenta('gulp [ production ] build:website'));
   gutil.log('Build the asset-pipeline and the website using Hugo');
```